src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -1076,6 +1076,11 @@ def import_certificate(cmd, client, vault_base_url, certificate_name, certificat
             secret_props['content_type'] = content_type
         elif certificate_policy and not secret_props:
             certificate_policy['secret_properties'] = SecretProperties(content_type=content_type)
+
+        attributes = certificate_policy.get('attributes')
+        if attributes:
+            attributes['created'] = None
```

Comment:
why set to None? rest patch doesn't allow passing those properties?

Reply:
> why set to None? rest patch doesn't allow passing those properties?

@yungezz Yes they are readonly attributes and they are always set as None in SDK:
![image](https://user-images.githubusercontent.com/3250203/82772942-5607c980-9e73-11ea-9738-bc19a1688ab4.png)
Although it is useless to pass value for them, the validator always works: it will check if the data type matches `unix-time`, so I planed to pass `None` for them. BTW, remove these 2 attributes is another way to do it.